sdcm/tester.py

```diff
@@ -762,6 +757,33 @@ def clean_resources(self):
         if self.monitors is not None:
             self.monitors.get_backtraces()
             self.monitors.download_monitor_data()
+
+            # upload prometheus data
+            prometheus_folder = glob.glob(os.path.join(self.logdir, '*monitor*'))[0]
```

Comment:
The monitor log directory is matched, why not directly match prometheus directory?

prometheus_folder = glob.glob(os.path.join(self.logdir, '*monitor*/prometheus/'))[0]